raiden-cli/src/index.ts

```diff
@@ -368,6 +373,19 @@ async function checkDisclaimer(accepted?: boolean): Promise<void> {
   assert(accepted, 'Disclaimer not accepted!');
 }
 
+async function getUdcAddressIfUnstable(
+  ethRpcEndpoint: string,
+  developmentEnvironment: string,
+): Promise<string | undefined> {
+  const provider = new JsonRpcProvider(ethRpcEndpoint);
+  const network = await provider.getNetwork();
+
+  if (network.name === 'goerli' && developmentEnvironment === 'unstable') {
+    return '0xD145252daB9323F9Acb70295d6a0589c611F225d';
```

Comment:
Wouldn´t this better go in `config.json`, and then you just use it if the `developmentEnvironment` is unstable?